### PR TITLE
Show entity bounding box for bad URLs

### PR DESCRIPTION
**Pull request by Atlante45**
- Show entities bounding box if the model failed to load.

## Test Plan:
- Set bad URL on a model entity
- Verify that you can see said entity's bounding box
- Verify you can still select it with your mouse/controllers

[Bug](https://highfidelity.fogbugz.com/f/cases/2083/Message-in-Entity-List-that-says-something-failed-to-load)

### CODE REVIEW

File: libraries/entities-renderer/src/RenderableModelEntityItem.cpp [hunk 1]
```diff
@@ -371,116 +371,117 @@ void RenderableModelEntityItem::render(RenderArgs* args) {
...
+        static glm::vec4 greenColor(0.0f, 1.0f, 0.0f, 1.0f);
+        gpu::Batch& batch = *args->_batch;
+        bool success;
+        auto shapeTransform = getTransformToCenter(success);
+        if (success) {
+            batch.setModelTransform(shapeTransform); // we want to include the scale as well
+            DependencyManager::get<GeometryCache>()->renderWireCubeInstance(batch, greenColor);
+        }
+        return;
+    }
```
Comment (Atlante45) [inline libraries/entities-renderer/src/RenderableModelEntityItem.cpp]: I just pulled the `else` clause up here.
All the changes beneath are the diff tool not liking big blocks of code being shifted over.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 09bee293e89dc2c71536417fc9efc329b5bed2cf available at: https://gist.github.com/e35329e7ef53f80f5965b13c15ca1948<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9991-09bee29.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9991-09bee29.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 4f8d9b190b4a8f2d04409c4b953e46cf5929dda0 available at: https://gist.github.com/e35329e7ef53f80f5965b13c15ca1948<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9991-4f8d9b1.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9991-4f8d9b1.dmg

**Comment by OmegaHeron** [comment]
Tested Windows 10
Desktop using keyboard/mouse.  Pass.  Placing a model with bad source (tried http and atp) showed a bounding box with mixed red/green lines.  When model deselected it turns to all green lines and when selected again, all red.

HMD (DK2) using xbox control was able to achieve same as above.

**Pull request merged**